src/event.rs

```diff
@@ -113,10 +113,10 @@ pub enum PressedState {
 
 impl PressedState {
     #[cfg_attr(any(feature = "test", test), allow(dead_code))]
-    fn from_state(state: glutin_event::ElementState) -> PressedState {
+    fn from_state(state: ElementState) -> PressedState {
         match state {
-            glutin_event::ElementState::Pressed => PressedState::Pressed,
-            glutin_event::ElementState::Released => PressedState::Released,
+            ElementState::Pressed => PressedState::Pressed,
+            ElementState::Released => PressedState::Released,
```

Comment:
Could you change this code back to the way it was? It looks like this isn't related to any changes needed when updating the glutin version.

Some background: When we initially wrote this code, we avoided importing too many `glutin::event` types because that helps clarify which types are coming from glutin and which types belong to our event API. Prefixing glutin's types with `glutin_event::` makes it completely unambiguous when you're reading the code. It would be great if the code could go back to using the convention.

Reply:
Okay sure